docs/UserGuide.md

```diff
@@ -610,10 +567,10 @@ This section contains features intended for advanced users.
 
 ### Editing the data file
 
-PropertyWhiz's data are saved as a JSON file `[JAR file location]/data/propertywhiz.json`. Advanced users are welcome to update data directly by editing that data file.
+PropertyWhiz's data are saved as a JSON (JavaScript Object Notation) file `[JAR file location]/data/propertywhiz.json`. Advanced users are welcome to update data directly by editing that data file.
```

Comment:
Glossary entry for JSON perhaps?

Reply:
I think as its only used once its ok to put it here